1038-vertical-nav-design-guidance.md

```diff
@@ -0,0 +1,35 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-21
+---
+
+# Vertical Nav Component
+
+[Vertical Nav](https://clarity.design/documentation/vertical-nav) is a vertically aligned navigational component. It can fit as many navigation links as needed, scrolling when the content exceeds the viewport.
```

Comment:
```suggestion
The [vertical nav](https://clarity.design/documentation/vertical-nav) component provides vertically-aligned navigation links. It can fit as many navigation links as needed by scrolling when the content exceeds the viewport.
```